### DISCUSSION TITLE
How to generate speciffic ore from oraxen in speciffic world?

### DISCUSSION THREAD

**Discussion by K0neki** [DerFrZocker/Custom-Ore-Generator#36] [Q&A]
Hi, i want to generate ore "orax_ore" with custom_model_data: 1004 in world_the_end. I just installed this plugin and i don't know how to create this things. I want those ores to spawn below y=15 and replace end_stone with 2-3 veins. Can someone help me?

**Comment by DerFrZocker** [comment]
To generated a Oraxen block follow the following steps:

1. Place the custom block
2. Use `/oregen create`
3. Enter a name like `MyCustomOre`
4. Left click the custom block you placed
5. Type `!next`
6. Left click the blocks you want to replace such as `stone` and `deepslate`
7. Type `!next`
8. Type `!next` again
For normal world generation:
9.  Select `Vanilla Minable Generator` (diamond block)
10. Select `Count Range` (iron block)
11. When it should generated in all biomes click `next` (stick)
12. Set a `vein size` (coal) and click `next` (stick)
13. Set a `height range`, `veins per chunk` and `minimum height` (coal) and click `next` (stick)
14. Click on `Oraxen`
15. Click on `Click to set the found Custom Data` (enchantet bock)
16. Click `next` (stick)
17. Click the world in which the ore should generated
18. Click `next` (stick)
19. Click `Click to create the ore-config` (upper green block)

If your Oraxen block does not generated in new chunks, then there might be an error with you configuration. In this case please send the configuration file found in the directory: `plugins\custom-ore-generator\data\ore-config`

**Reply by K0neki** [reply]
do you have a solution?

**Comment by K0neki** [comment]
I think ores doesn't generate
I wanted them to generate on the bottom of the island to make players build a little bit down to the void to get ores

# Only edit this file if you 10000% sure you know what you are doing. 
# You can set everything via the plugin commands, no need to edit this file manully.

value:
  ==: CustomOreGenerator#OreConfig
  name: Ender_ruda
  material: NOTE_BLOCK
  ore-generator: SINGLE_ORE_GENERATOR
  block-selector: HIGHEST_BLOCK
  activated: true
  generated-all: false
  biomes:
  - SMALL_END_ISLANDS
  - END_BARRENS
  - END_HIGHLANDS
  - END_MIDLANDS
  replace-materials:
  - END_STONE
  custom-data:
    ORAXEN: orax_ore
